src/styles/variable.css

```diff
@@ -28,21 +28,21 @@
   --shadow-lg: rgba(0, 0, 0, 0.15);
 
   /* Border */
-  --border-sm: 1px;
-  --border-lg: 1.5px;
-  --border-xl: 2px;
+  --border-sm: 0.0625rem; /* 1px */
+  --border-lg: 0.09375rem; /* 1.5px */
+  --border-xl: 0.125rem; /* 2px */
 
   /* Border radius */
-  --border-radius: 10px;
+  --border-radius: 0.625rem; /* 10px */
 
-  /*Light height */
-  --line-height-sm: 19.2px;
-  --line-height-md: 24px;
-  --line-height-lg: 30px;
-  --line-height-xl: 75px;
+  /* Line height */
+  --line-height-sm: 1.2rem; /* 19.2px */
```

Comment:
I don't think these line-height correct, let show me the image where you will use this values.

Reply:
![image](https://github.com/thanhtrantruong47/react-training/assets/155411536/505d6e6c-1490-4c92-aa25-eb7a0d98e067) 
![image](https://github.com/thanhtrantruong47/react-training/assets/155411536/5007bff3-3d0b-4a8a-b1ec-4f46b2abfdb3)
![image](https://github.com/thanhtrantruong47/react-training/assets/155411536/89c02137-697b-494d-abec-20bfdd48af4b)
![image](https://github.com/thanhtrantruong47/react-training/assets/155411536/2daae2c9-991f-43ce-befa-180b8c1a8666)
![image](https://github.com/thanhtrantruong47/react-training/assets/155411536/26404d87-79f3-43cb-9337-0f81307321ae)
![image](https://github.com/thanhtrantruong47/react-training/assets/155411536/a4827434-9b2e-45db-b7cb-2fd33b4a7259)